readthedocs/audit/serializers.py

```diff
@@ -0,0 +1,75 @@
+"""
+Serializers used to save the corresponding data from a model in a log entry.
+
+.. note::
+
+   These aren't used in an API, but to extract data from objects to
+   save it in log entries (AuditLog.data).
+"""
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from readthedocs.invitations.models import Invitation
+from readthedocs.organizations.models import Organization, Team
+from readthedocs.projects.models import Project
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["id", "username"]
+
+
+class OrganizationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Organization
+        fields = ["id", "slug"]
+
+
+class TeamSerializer(serializers.ModelSerializer):
+
+    organization = OrganizationSerializer()
+
+    class Meta:
+        model = Team
+        fields = ["id", "slug", "organization"]
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+
+    organization = OrganizationSerializer()
+
+    class Meta:
+        model = Project
+        fields = ["id", "slug", "organization"]
+
+
+class InvitationSerializer(serializers.ModelSerializer):
+
+    """Invitation serializer."""
+
+    to_user = UserSerializer()
+    from_user = UserSerializer()
+    object = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Invitation
+        fields = [
+            "id",
+            "from_user",
+            "to_user",
+            "to_email",
+            "object_type",
+            "object",
+            "from_user",
+        ]
+
+    # pylint: disable=no-self-use
+    def get_object(self, obj):
+        serializers = {
+            "organization": OrganizationSerializer,
+            "project": ProjectSerializer,
+            "team": TeamSerializer,
```

Comment:
It would be good to have a comment here saying that the value of `object` will depend on the object type and a different serializer will be used.